circle.yml

```diff
@@ -4,4 +4,4 @@ dependencies:
 test:
   override:
     - npm test
-    - ./node_modules/.bin/grunt shell:coveralls
+    - ./node_modules/.bin/grunt shell:coveralls --force
```

Comment:
Oh, I was wondering about this. Were you running into errors with coveralls? Now that I think about it, that may bomb if it's not run under the NREL CircleCI account. So does this fix things for you, or is there anything else you think we should do to clean this up?